### PR TITLE
Introduce commit

### PR DESCRIPTION
**Pull request by gigbat**

### CODE REVIEW

File: src/main/java/com/dev/cinema/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    @Override
+    public String toString() {
+        return "Order - [id = " + id
+                + ", tickets = " + tickets
+                + ", user = " + user + "]";
```
Comment (vlykhvar) [inline src/main/java/com/dev/cinema/model/Order.java]:
```suggestion
        return "Order - [id = " + id
                + ", tickets = " + tickets
                + ", orderDate=" + orderDate
                + ", user = " + user + "]";
```

Comment (vlykhvar) [reply src/main/java/com/dev/cinema/model/Order.java]: I think customer should know when he/she made an order

**Review by vlykhvar — APPROVED** [review]

**Review by vlykhvar — APPROVED** [review]

**Review by vlykhvar — APPROVED** [review]

File: src/main/java/com/dev/cinema/model/Order.java (hunk 1 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/Order.java]: okey, done

File: src/main/java/com/dev/cinema/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+@Entity
+@Table (name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @OneToOne
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/model/Order.java]: are you sure here?

File: src/main/java/com/dev/cinema/model/Order.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    public void setUser(User user) {
+        this.user = user;
+    }
+
+    @Override
+    public String toString() {
+        return "Order - [id = " + id
+                + ", tickets = " + tickets
+                + ", oder date = " + orderDate
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/model/Order.java]:
```suggestion
                + ", order date = " + orderDate
```

File: src/main/java/com/dev/cinema/service/impl/OrderServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        order.setUser(shoppingCart.getUser());
+        order.setTickets(new ArrayList<>(shoppingCart.getTicketList()));
+        orderDao.add(order);
+        shoppingCartService.clear(shoppingCart);
+        return order;
+    }
+
+    @Override
+    public List<Order> getOrdersHistory(User user) {
+        ShoppingCart cartUser = shoppingCartService.getByUser(user);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/service/impl/OrderServiceImpl.java]: do you need this?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/model/Order.java (hunk 2 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/Order.java]: fixed

File: src/main/java/com/dev/cinema/model/Order.java (hunk 3 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/Order.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**